master/internal/user/postgres_agentusergroup.go

```diff
@@ -56,7 +57,9 @@ func GetAgentUserGroup(
 	workspaceID int,
 ) (*model.AgentUserGroup, error) {
 	workspaceAug, err := getAgentUserGroupFromWorkspaceID(ctx, workspaceID)
-	if err != nil {
+	if err == sql.ErrNoRows {
+		logrus.WithError(err).Warnf("no agent user group results from workspaceID=%d", workspaceID)
+	} else if err != nil {
```

Comment:
`workspaceID` is supposed to be optional, but bun throws an error if zero results are returned.